config/owasp/suppressions.xml

```diff
@@ -1,11 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress until="2030-01-01">
-    <notes><![CDATA[
-     Suppressing as it's a false positive (see: https://pivotal.io/security/cve-2018-1258)
-   ]]></notes>
-    <gav regex="true">.*</gav>
-    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+  <suppress>
+    <notes>groovy-all</notes>
+    <cve>CVE-2020-17521</cve>
+  </suppress>
+  <suppress>
+    <notes>spring-boot-starter-oauth2-client</notes>
+    <cve>CVE-2018-1258</cve>
+    <cve>CVE-2021-22112</cve>
```

Comment:
@Gareth40342 I have created this PR. please could you check if the pipeline dependancy vulnerability checks pass for this PR.